src/main/java/courseitda/common/exception/ErrorCode.java

```diff
@@ -9,10 +9,227 @@
 @RequiredArgsConstructor
 public enum ErrorCode {
 
+    /*
+     * 0000 Series: General Errors
+     * 1000 Series: Authentication and Authorization Errors
+     * 2000 Series: Workspace Errors
+     * 3000 Series: Category Errors
+     * 4000 Series: Category Place Errors
+     * 5000 Series: Member Errors
+     * */
+
+    // TEMPORARY_ERROR - 0000
     TEMPORARY_ERROR(
             "0000",
             "커스텀 예외로 전환된 임시 에러입니다.",
-            HttpStatus.INTERNAL_SERVER_ERROR
+            HttpStatus.INTERNAL_SERVER_ERROR // 500
+    ),
+
+    // 1000 Series: Authentication and Authorization Errors
+    MISSING_AUTH_HEADER(
+            "1001",
+            "Authorization 헤더가 존재하지 않습니다.",
+            HttpStatus.UNAUTHORIZED // 401
+    ),
+
+    MALFORMED_BEARER_TOKEN(
+            "1002",
+            "Bearer 토큰 형식이 아닙니다.",
+            HttpStatus.UNAUTHORIZED // 401
+    ),
+
+    EXPIRED_OR_INVALID_TOKEN(
```

Comment:
`INVALID`(유효하지 않은) 의미에 `EXPIRED`(만료됨)도 포함되어 있어서, 다음과 같이 네이밍을 변경해보는건 어떨까요!

```suggestion
    INVALID_TOKEN(
```